anti-analysis/anti-vm/vm-detection/reference-anti-vm-strings---wmi-disk-drive-model-contains-virtual.yml

```diff
@@ -0,0 +1,19 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: reference anti-VM strings - WMI disk drive model contains Virtual
+    namespace: anti-analysis/anti-vm/vm-detection
+    authors:
+      - anders.vejlby@mandiant.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Virtualization/Sandbox Evasion::System Checks [T1497.001]
+    references:
+      - <insert_references>
+    examples:
+      - E69F2E964AA4A4E676F3335705B047E7:0x4035e0
+  features:
+    - or:
+      - string: "Win32_DiskDrive"
+      - string: "Model"
+      - string: "Virtual"
```

Comment:
is it reasonable to also include logic around WMI query execution? or is that code found in a different function?